src/plugins/formvalid/_base.scss

```diff
@@ -46,5 +46,31 @@
 				@extend %formvalid-error-label;
 			}
 		}
+		&.h5 {
+			float: none;
+			margin-bottom: 0;
+			.error {
+				margin-top: 1em;
+			}
+		}
 	}
 }
+ul.form-group {
```

Comment:
This style do not belong to the formvalid plugin, instead it should be more global, like the bootstrap override